Documentation/SettingsOverview.md

```diff
@@ -0,0 +1,298 @@
+- **S.solver.nlpsol_options**:
+    - Options to be passed to [CasADi's nonlinear program solver](https://web.casadi.org/api/html/d4/d89/group__nlpsol.html). Default is *[]* [struct].
+- **S.solver.ipopt_options**:
+    - Options to be passed to [ipopt](https://coin-or.github.io/Ipopt/OPTIONS.html#OPTIONS_REF). Default is *[]* [struct]. Overwrites S.solver.nlpsol_options.ipopt.
+- **S.solver.linear_solver**: 
+    - linear solver algorithm used by ipopt. Default is *mumps* [char]. Overwrites S.solver.ipopt_options.linear_solver.
+- **S.solver.tol_ipopt**: 
+    - the [ipopt convergence tolerance](https://coin-or.github.io/Ipopt/OPTIONS.html#OPT_tol) is set to 10^(-S.solver.tol_ipopt). A higher number gives a more precise answer, but takes more time. Default is *4* [double]. Overwrites S.solver.ipopt_options.tol.
+- **S.solver.constr_viol_tol_ipopt**: 
+    - the [ipopt constraint violation tolerance](https://coin-or.github.io/Ipopt/OPTIONS.html#OPT_constr_viol_tol) is set to 10^(-S.solver.constr_viol_tol_ipopt). A higher number gives a more precise answer, but takes more time. Default is *6* [double]. Overwrites S.solver.ipopt_options.constr_viol_tol.
+- **S.solver.max_iter**: 
+    - maximal amount of iterations after wich the solver will stop. Default is *10000* [double]. Overwrites S.solver.ipopt_options.max_iter.
+
+#### S.subject
+
+- **S.subject.mass**: 
+    - mass of the subject in kilograms. Default is *[]* kilograms [double]. Default is empty, it will be overwritten by the mass extracted from the OpenSim model.
+- **S.subject.IG_pelvis_y**: 
+    - height from ground to pelvis, in meters. Default is *[]* m [double]. Default is empty, it will be overwritten by pelvis height extracted from the OpenSim model.
+    - always used for the quasi-random initial guess
+    - used for data-informed initial guess when `S.subject.adapt_IG_pelvis_y = 1;`
+    - S.subject.IG_pelvis_y is also used to establish bounds on vertical pelvis position.
+- **S.subject.adapt_IG_pelvis_y**: 
+    - boolean to adjust the trajectory of height of pelvis from the ground for data-informed initial guess. Default is *0*. 0 means the trajectory will not be changed. If 1, the trajectory will be changed such that the average value of the trajectory is equal to S.subject.IG_pelvis_y.
+- **S.subject.muscle_strength**: 
+    - structure with [scaling factors for muscle strength](./FiguresForDocumentation/fig_muscle_tendon_properties_scaling.png). This scales the max muscle force of the active muscle force. Default is *[]*, that is, no scaling. Input as a cell array where 1st input is the muscle(s) name, 2nd is the scale factor. If more than one scaling is to be performed, add 2 more inputs. For example, S.subject.muscle_strength = {{'soleus_l','soleus_r'},0.9,{'tib_ant_l'},1.1} will scale both soleus by a factor of 0.9 and tibialis anterior left by a scale of 1.1.
```

Comment:
figure is missing in the folder

Reply:
oops! Updated the gitignore, now the figures should be there.